README.md

```
@@ -177,51 +139,40 @@
+    - Git
+
+## Instalación y Uso
+
+- Requisitos Previos
+    - Node.js 14 o superior.
+    - npm o yarn instalado.
+
+- Pasos para Ejecutar el Proyecto
+    - Clone el repositorio:
+    git clone https://github.com/tu_usuario/assetmaker.git
+    - Instale las dependencias:
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)